src/client/app/styles/spinner.css

```diff
@@ -2,6 +2,7 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+/* Creates general spinner visuals, used and modified by spinner-item-1 and spinner-item-2 */
```

Comment:
As a declarative element, I would swap "creates" for "specifies", or simply remove it: "General spinner visuals..."